### DISCUSSION TITLE
Implicit return

### DISCUSSION THREAD

**Discussion by abersheeran** [hsfzxjy/lambdex#2] [Ideas]
In Rust functions, the last expression will be returned as the function return value, without the need to use the return keyword. Are you considering including the same features in this project?

**Comment by hsfzxjy** [comment]
Good suggestion. I am going to add it as an optional feature.

**Comment by hsfzxjy** [comment]
The feature is now available as an extension, plus an optional Rust-style await feature. Please check https://github.com/hsfzxjy/lambdex#language-extension for more details.
